recipes/cosmosis-build-standard-library/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "cosmosis-build-standard-library" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url:  https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d818aa4f0bebf3d5507014a7042caf1f39636215a348322129647e51fa874055
+
+build:
+  number: 0
+  skip: True  # [(py < 36) or win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - cosmosis
+    # For compiling standard librarymodules
+    - compilers
+    - {{ mpi }}
```

```suggestion
    - {{ mpi }}
    - llvm-openmp  # [osx]
    - libgomp  # [linux]
```

I think you may need these when building the standard library. I see `-fopenmp` flags without these libs.

---

It appears these libs are getting pulled in anyways, but they are direct dependencies and so need to be listed explicitly.